CHANGELOG.md

```diff
@@ -5,6 +5,35 @@ The original MRs are only visible on the [LIGO GitLab repository](https://git.li
 
 ## [Unreleased]
 
+## [2.6.0]
+
+We noticed a longstanding bug in the `GravitationalWaveTransient` likelihood that caused the PSD to be
```

Comment:
We should clarify this is a misunderstanding in the field rather than a bug that only impacts bilby